templates/checkout/snippets/addresses-form-panel.html

```diff
@@ -1,12 +1,15 @@
-<div class="radio {% if address or label == _('Same as shipping') %} address_hide{% else %} address_show{% endif %}">
+ {% load i18n_address_tags %}
+ <div class="radio {% if address or label == _('Same as shipping') %} address_hide{% else %} address_show{% endif %}">
```

Comment:
Please open an issue to get rid of `label == _('Same as shipping')`.

Reply:
The issue no. is #1585